### PR TITLE
CoC transparency report 2022

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

File: content/blog/2023-01-06-transparency2022/index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+The rOpenSci community is supported by our [Code of Conduct](https://ropensci.org/code-of-conduct) with a clear description of unacceptable behaviors, 
+instructions on how to make a report, and information on how reports are handled. We, the Code of Conduct Committee, 
+are responsible for receiving, investigating, deciding, enforcing and reporting on all reports of potential 
+violations of our Code. We are committed to transparency with our community while upholding the privacy 
+of people who experience or report incidents.
+
+In 2022, we received one Code of Conduct incident report. 
+
+### Summary of Reported Incidents
```
Comment (steffilazerte) [inline content/blog/2023-01-06-transparency2022/index.md:24]:
```suggestion
### Summary of reported incident
```

File: content/blog/2023-01-06-transparency2022/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+* _Description of the incident:_ During a package review process, a reviewer became extremely negative on a public 
+channel and posted messages publicly shaming the package's authors with non-constructive feedback and comments.
```
Comment (steffilazerte) [inline content/blog/2023-01-06-transparency2022/index.md:27]:
```suggestion
* _Description of the incident:_ During a package review process, a reviewer was extremely negative on a public 
channel and posted messages publicly shaming the package's authors with non-constructive feedback and comments.
```

File: content/blog/2023-01-06-transparency2022/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+* _Unacceptable behavior:_ public shaming and not complying with "be considerate in speech and actions."
+* _Community space:_ GitHub
+* _Actions:_  We follow our protocols to investigate the incident and inform the reported and reporter of our decision and the outcome. 
+We reinforce on the communication that such comments are not acceptable under our CoC. We also changed the 
+text of the CoC to include more examples from online spaces and suggested a change in the review guide to be 
+more explicit about non-constructive feedback.
```
Comment (steffilazerte) [inline content/blog/2023-01-06-transparency2022/index.md:33]:
```suggestion
* _Actions:_  We followed our protocols to investigate the incident and informed the reported and reporter of our decision and the outcome. 
We reinforced in communications that such comments are not acceptable under our CoC. We also changed the 
text of the CoC to include more examples from online spaces and suggested a change in the review guide to be 
more explicit about non-constructive feedback.
```

File: content/blog/2023-01-06-transparency2022/index.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+* _Observations:_ the reporter expressed that the reported person apologizes and doesn't require further actions. 
+The reporter also suggested changes to the review guide and the CoC, that we have taken into account through changes described above.
```
Comment (steffilazerte) [inline content/blog/2023-01-06-transparency2022/index.md:35]:
```suggestion
* _Observations:_ the reporter relayed that the reported apologized and that they do not require further actions. 
The reporter also suggested changes to the review guide and the CoC, which we have taken into account through changes described above.
```

File: content/blog/2023-01-06-transparency2022/index.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+
+This code of conduct belongs to the community. We welcome your feedback at any time by emailing conduct at ropensci.org.
```
Comment (steffilazerte) [inline content/blog/2023-01-06-transparency2022/index.md:38]: See my rational for why in the CoC update
```suggestion
This code of conduct belongs to the community. We welcome your feedback at any time by emailing <conduct@ropensci.org>.
```

File: content/blog/2023-01-06-transparency2022/index.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  - Kara Woo
+date: 2023-01-06
+tags:
+  - community
+  - code of conduct
+  - governance
+description: rOpenSci 2022 Code of Conduct Transparency Report
+---
+
+The rOpenSci community is supported by our [Code of Conduct](https://ropensci.org/code-of-conduct) with a clear description of unacceptable behaviors,
```
Comment (steffilazerte) [inline content/blog/2023-01-06-transparency2022/index.md:16]:
```suggestion
The rOpenSci community is supported by our [Code of Conduct](/code-of-conduct) with a clear description of unacceptable behaviors, 
```

**Review by steffilazerte — COMMENTED** [review]
Looks good! 

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [X] alternative text of images is informative
* [X] ~~Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")~~
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)

**Review by mpadge — APPROVED** [review]
That looks great, thanks @yabellini

**Pull request merged**